### PR TITLE
Scale track markers appropriately when zooming on track map

### PR DESCRIPTION
**Pull request by brentpicasso**
dynamically adjust size of track markers if trackmap is zoomed.

### CODE REVIEW

**Comment by brentpicasso** [comment]
resolves #755

File: main.py [hunk 1]
```diff
@@ -283,15 +283,12 @@ def on_stop(self):
...
-        try:
-            view = self.mainViews.get(view_name)
-            if not view:
-                view = self.view_builders[view_name]()
-                self.screenMgr.add_widget(view)
-                self.mainViews[view_name] = view
-            self.screenMgr.current = view_name
-        except Exception as detail:
-            Logger.error('RaceCaptureApp: Failed to load main view ' + str(view_name) + ' ' + str(detail))
+        view = self.mainViews.get(view_name)
```
Comment (brentpicasso) [inline main.py]: mostly unrelated refactor: don't swallow exceptions; let crash handler catch it.

**Comment by ryandoherty** [comment]
:shipit:
